### PR TITLE
docs: encrypt CLI usage

### LINKED ISSUE #21157: Document CLI encrypt
### What would you like Renovate to be able to do?

Document how to encrypt secrets using CLI only

### If you have any ideas on how this should be implemented, please tell us here.

```
curl https://app.renovatebot.com/renovate.pgp --output renovate.pgp
echo -n '{"o":"your-organization", "r":"your-repository (optional)", "v":"your-secret-value"}' | jq . -c | gpg --encrypt -a --recipient-file renovate.pgp | grep -v '^----' | tr -d '\n'
```

### Is this a feature you are interested in implementing yourself?

Maybe

### PR DESCRIPTION
**Pull request by rarkins**
## Context

Closes #21157

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — DISMISSED** [review]

File: docs/usage/configuration-options.md [hunk 1]
```diff
@@ -813,6 +828,8 @@ See [Private module support](https://docs.renovatebot.com/getting-started/privat
 
 Encrypted secrets typically have a single org, but you may encrypt a secret with more than one, e.g. specifying `org1,org2` to allow the secret to be used in both `org1` and `org2` organizations.
 
+For additional information about secrets use for private packages you can also read [Private package support](https://docs.renovatebot.com/getting-started/private-packages).
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:831]:
```suggestion
For more information on how to use secrets for private packages, read [Private package support](https://docs.renovatebot.com/getting-started/private-packages).
```

File: docs/usage/configuration-options.md [hunk 2]
```diff
@@ -802,9 +802,24 @@ For the full list of available managers, see the [Supported Managers](https://do
 
 ## encrypted
 
-Use this to encrypt secrets in a way which can be stored in repository configs.
+Use this to encrypt secrets in a way which can be stored in repository configs. It is possible to encrypt secrets using either a HTML page, or CLI.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:805]:
```suggestion
Before you put any secrets in your repository configuration, encrypt the secrets.
You can encrypt secrets using either a HTML page, or the CLI.
```

File: docs/usage/configuration-options.md [hunk 3]
```diff
@@ -802,9 +802,24 @@ For the full list of available managers, see the [Supported Managers](https://do
...
 
-See [Private module support](https://docs.renovatebot.com/getting-started/private-packages) for details on how this is used to encrypt npm tokens.
+To encrypt secrets for the hosted Mend Renovate app for github.com using a HTML page, go to [app.renovatebot.com/encrypt](https://app.renovatebot.com/encrypt) and complete the form. Alternatively, this form may be downloaded and edited for self-hosted users to use their own PGP public key.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:807]:
```suggestion
To encrypt secrets for the hosted Mend Renovate app for github.com with a HTML page, go to [app.renovatebot.com/encrypt](https://app.renovatebot.com/encrypt) and complete the form.
If you're self-hosting Renovate, you may download and edit the form, to use your own PGP public key.
```

File: docs/usage/configuration-options.md [hunk 4]
```diff
@@ -802,9 +802,24 @@ For the full list of available managers, see the [Supported Managers](https://do
...
+
+You can also encrypt secrets using the CLI, using the `gpg` command line tool plus some help from `curl` and `jq`. Here is an example:
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:809]:
```suggestion
You can also encrypt secrets from the CLI, using the `curl`, `echo`, `jq`, `gpg`, `grep` and `tr` CLI programs.
Here is an example:
```

File: docs/usage/configuration-options.md [hunk 5]
```diff
@@ -802,9 +802,24 @@ For the full list of available managers, see the [Supported Managers](https://do
...
+echo -n '{"o":"your-organization", "r":"your-repository (optional)", "v":"your-secret-value"}' | jq . -c | gpg --encrypt -a --recipient-file renovate.pgp | grep -v '^----' | tr -d '\n'
+```
+
+The above script does this:
+
+- Uses `curl` to download the Mend Renovate hosted app's public key
+- Echos a JSON object into `jq` for validation and compacting
+- Uses `gpg` to encrypt the contents
+
+The `jq` step is optional, you could leave it out if you wish. Its primary value is validating that the string you input to `gpg` is valid JSON, and compact.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:822]:
```suggestion
The above script uses:

- `curl` to download the Mend Renovate hosted app's public key
- `echo` to echo a JSON object into `jq`
- `jq` to validate the JSON and then compact it
- `gpg` to encrypt the contents
- `grep` to ...
- `tr` to ...

The `jq` step is optional, you can leave it out if you wish.
Its primary value is validating that the string you echo to `gpg` is valid JSON, and compact.
```

Before you apply this suggestion, fill out the `...` parts to explain what that part of the command does. 😉

File: docs/usage/configuration-options.md [hunk 6]
```diff
@@ -813,6 +828,8 @@ See [Private module support](https://docs.renovatebot.com/getting-started/privat
 
 Encrypted secrets typically have a single org, but you may encrypt a secret with more than one, e.g. specifying `org1,org2` to allow the secret to be used in both `org1` and `org2` organizations.
```
Comment (HonkingGoose) [inline docs/usage/configuration-options.md:829]:
```suggestion
Encrypted secrets usually have a single organization.
But you may encrypt a secret with more than one organization, for example: `org1,org2`.
This way the secret can be used in both the `org1` and `org2` organizations.
```

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
Changes:

- Use simple words and phrases
- List all CLI commands used in the example
- Explain all CLI commands (needs maintainer input 😉)
- Fully expand the `org` abbreviation into the word `organization(s)`

**Review by viceice — APPROVED** [review]

**Pull request merged**